Introduce kyma-developer role (#2782)
kyma-project/kyma: docs/security/docs/08-02-manage-static-users.md

## Bind a user to a Role or a ClusterRole

A newly created static user has no access to any resources of the cluster as there is no Role or ClusterRole bound to it.  
By default, Kyma comes with the following ClusterRoles:
 

> - **kyma-admin**: allows to get full admin access to the entire cluster

Reviewer: ```suggestion
- **kyma-admin**: allows full admin access to the entire cluster
```

> - **kyma-edit**: allows full access to kyma managed resources

Reviewer: ```suggestion
- **kyma-edit**: allows full access to all Kyma-managed resources
```

> - **kyma-developer**: **kyma-edit** + full access to base kubernetes resources

Reviewer: ```suggestion
- **kyma-developer**:  allows full access to Kyma-managed resources and basic Kubernetes resources
```

Reviewer: I think you should list these Kubernetes resources here. 
`(...)and basic Kubernetes resources, such as (...)`

> - **kyma-view**: allows to view and list all of the resources of the cluster.

Reviewer: ```suggestion
- **kyma-view**: allows to view and list all of the resources of the cluster
```
